src/main.cpp

```diff
@@ -1,76 +1,68 @@
 #include <Arduino.h>
 #include <dataqueue.h>
-#include <display.h>
+#include <led.h>
 #include "ap.h"
 #include "gateway.h"
 #include "loramesh.h"
+#include "monitor.h"
+#include "wifinode.h"
 
-#define BLUE_LED 2
-
-#define OLED_SDA 4
-#define OLED_SCL 15
-#define OLED_RST 16
-#define OLED_SCREEN_WIDTH 128
-#define OLED_SCREEN_HEIGHT 64
+// Observability defines
+#ifndef MONITOR_CLASS
+#define MONITOR_CLASS LogMonitor
+#endif
+#ifndef MONITOR_UPDATE_INTERVAL
+#define MONITOR_UPDATE_INTERVAL 1000
+#endif
+#ifndef LED_TO_BLINK
+#define LED_TO_BLINK -1
+#endif
```

Comment:
This naming is weird, just "BLINK" or "BLINKI_LED" reads better. We can also add other effects like BREATH_LED or QUICK_BLINK_LED later.